docs/sourcegraph-3.0.migrate.md

```diff
@@ -7,3 +7,7 @@
 ## Language server deployment
 
 Sourcegraph 3.0 removed lsp-proxy and automatic language server deployment in favor of [Sourcegraph extensions](https://docs.sourcegraph.com/extensions). As a consequence, Sourcegraph 3.0 does not automatically run or manage language servers. If you had code intelligence enabled in 2.x, you will need to follow the instructions for each language extension and deploy them individually. Read the [code intelligence documentation](https://docs.sourcegraph.com/user/code_intelligence).
+
+## HTTPS / TLS
+
+Sourcegraph 3.0 removed HTTPS / TLS features from Sourcegraph in favor of relying on [Kubernetes Ingress Resources](https://kubernetes.io/docs/concepts/services-networking/ingress/). As a consequence, Sourcegraph 3.0 does not expose TLS as the NodePort 30433. Instead you need to ensure you have setup and configured an ingress controller. See [ingress controller documentation](docs/configure.md#ingress-controller-recommended) and [configure TLS/SSL documentation](docs/configure.md#configure-tlsssl).
```

Comment:
Should we also document that you should remove `TLS_CERT` and `TLS_KEY` from the environment variables and site configuration?

Reply:
Yes! Will update.